.eslintrc.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
         __COMPONENTS__: true,
         __FUNDING_ELIGIBILITY__: true,
         __PAYPAL_DOMAIN__: true,
+        __FORCE_PAYPAL_DOMAIN_LOGGER__: true,
```

Comment:
New global variable to force logger into PayPal domain. Not sure how this library is deployed. I guess we can make this change for the China team. They have an odd integration case, only applicable to them.